src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -262,7 +264,7 @@ void uartIrqHandler(uartPort_t *s)
 
 static void handleUsartTxDma(uartPort_t *s)
 {
-    // DMA_Cmd(s->txDMAStream, DISABLE);
+    DMA_Cmd(s->txDMAStream, DISABLE);
```

Comment:
This may still abort transfer started from 'uartWrite'. It DMA stream is not disabled here, it will trigger another interrupt eventually